introduction.md

```diff
@@ -1,2 +1,5 @@
 My name is Jean Carlos. I am passionate for software development and learn new things. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?